### DISCUSSION TITLE
Is calibration necessary?

### DISCUSSION THREAD

**Discussion by ryneeverett** [deividAlfa/stm32_soldering_iron_controller#122] [Q&A] [answered]
I'm thinking of buying a Quicko T12-955 to use with this firmware.

Do I need to buy a tip thermometer as well or is calibration not strictly necessary? Are the knockoff tip thermometers any good? The brand name tip thermometers would seem to be as expensive as brand name soldering stations which would break the economic logic of buying a knockoff soldering station. So how do people handle this?

**Comment by deividAlfa** [comment, marked as answer]
Yes. I'm not KSGER or Quicko developer, nor I have access to every hardware out there, some versions showed considerable calibration differences (Not sure why, it's the same amplifier circuit, maybe diffferent amplifier ic causing input offset variation).
Better be safe than sorry, so the default calibration values are a bit low, you can get a cheap DMM with temperature probe in Aliexpress for $6 or so.
You don't need lab-grade equipment, it's just a soldering iron. A 10-20ºC error won't matter much.

**Comment by ryneeverett** [comment]
> You don't need lab-grade equipment, it's just a soldering iron. A 10-20ºC error won't matter much.

Thanks! This was unclear from the [calibration docs](https://github.com/deividAlfa/stm32_soldering_iron_controller/blob/master/Readme_files/Operation.md#calibration), which say:

> Requires a tip thermometer (e.g. Hakko FG-100 or similar).

**Comment by deividAlfa** [comment]
Maybe I should change that. The FG-100 is specifically made for soldering irons, but not expensive either, about $10-12.
A generic thermocouple probe will work good enough.

**Comment by ryneeverett** [comment]
>  The FG-100 is specifically made for soldering irons, but not expensive either, about $10-12.

You must be referring to the RST FG-100. I'm not seeing the Hakko FG-100 below $260.

**Reply by deividAlfa** [reply]
You can get a clone at aliexpress for $10 or so.
